bindings/tflite/java/org/tensorflow/lite/TensorFlowLite.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tensorflow.lite;
+
+/** Static utility methods loading the TensorFlowLite runtime. */
+public final class TensorFlowLite {
+  private static final Throwable LOAD_LIBRARY_EXCEPTION;
+  private static volatile boolean isInitialized = false;
+
+  static {
+    Throwable loadLibraryException = null;
+    try {
+      System.loadLibrary("iree");
```

Comment:
we'll want to call this something more specific to the tflite bindings - `iree_bindings_tflite` or something matching the code path is unambiguous (the only library we want to load here is the one with the JNI methods in it, and I could imagine `libiree.so` binaries that don't have them)